astro_2.0/components/DaoDetails/DaoDetailsGrid/DaoDetailsGrid.tsx

```diff
@@ -162,7 +162,9 @@ export const DaoDetailsGrid: FC<DaoDetailsGridProps> = ({
                 </ActionButton>
                 <ActionButton
                   iconName="proposalBounty"
-                  onClick={e => handleClick(e, `/dao/${id}/tasks/bounties`)}
+                  onClick={e =>
```

Comment:
I would suggest to use routing constants for navigation

Reply:
fixed